Source/Core/VideoCommon/CommandProcessor.h

```diff
@@ -155,7 +155,12 @@ union UCPClearReg
   UCPClearReg(u16 _hex) { Hex = _hex; }
 };
 
-u32 GetPhysicalAddressMask();
+inline constexpr u32 GetPhysicalAddressMask(bool is_wii)
```

Comment:
```suggestion
constexpr u32 GetPhysicalAddressMask(bool is_wii)
```

super minor, but constexpr already implies inline on functions